src/gc-cpu-runner/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+if(GC_LINK_DYNAMIC_LLVM)
+  set(LLVM_LINK_COMPONENTS
+    LLVM
+    )
+  set(MLIR_LINK_COMPONENTS
+    MLIR
+    MLIRExecutionEngine
+    MLIRJitRunner
+  )
+else()
+  if(NOT LLVM_LINK_LLVM_DYLIB)
+    set(LLVM_LINK_COMPONENTS
+      Core
+      Support
+      nativecodegen
+      native
+    )
+  endif()
+  set(MLIR_LINK_COMPONENTS
+    MLIRAnalysis
```

Comment:
Please add a comment about where to get this list.

Reply:
OK, done.